.github/actions/benchmark-build/benchmark-Windows.sh

```diff
@@ -4,13 +4,27 @@
 # 2. getting the cmake directory for running cpack with an absolute path (chocolatey has an unfortunately named alias)
 
 echo "Building ${CPACK_GEN} installer with ${BUILD_GEN} for ${BUILD_ARCH}"
-BOOST_ROOT="${BOOST_ROOT_1_72_0}"
+
```

Comment:
This looks pretty similar in all the files.  can we put it a script so it easier to change the next time we update boost?

Reply:
It should be safe to assume that the scripts are never run using a symlink?

Reply:
I think my biggest concern is a lot of duplication and having to update the boost version in quite a few spots.  If we can reduce that in some way I think would be good

Reply:
Boost installation is split into a shared script. The way I'm getting the path won't work if the script using the shared code is symlinked, but being located along a path with a symlinked directory should be okay.